util/recommender_configuer.py

```diff
@@ -39,6 +39,15 @@ def get_collaborative_filtering(self):
         """
         return self.config_dict['collaborative-filtering']
 
+    def get_recommender(self):
+        """
+        Get the recommender type
+
+        :returns: A string of userbased or itembased recommendations.
+        :rtype: str
```

Comment:
No, this is a string of 'itembased' or 'usedbased', based upon, we will use the results of contentbased or collaborative-filtering for the predictions.